app/src/main/java/me/saket/dank/ui/submission/SubmissionTitleSpan.kt

```diff
@@ -0,0 +1,16 @@
+package me.saket.dank.ui.submission
+
+import android.text.Spannable
+import me.saket.dank.widgets.TextViewLimitSelection
+
+// This is a no-op span to identify title position in TextView
+class SubmissionTitleSpan {
+  companion object {
+    @JvmStatic fun limitSelectionForTextView(view: TextViewLimitSelection) {
+      val title = view.text as Spannable
+      title.getSpans(0, title.length, SubmissionTitleSpan::class.java).take(1).forEach {
```

Comment:
`firstOrNull` would be a better choice than `take(1)` here. That way it will be clear that we are operating on a single span.